sheet/ozymandias/Aldin.qml

```diff
@@ -0,0 +1,501 @@
+import QtQuick 2.3
+
+import "../"
+import org.lasath.psychic_bear 1.0
+import "../shared/feats" as Feats
+
+Item {
+    AbilityScore {
+        id: strength
+        name: "Strength"
+
+        permanent.modifiers: [
+            Bonus {
+                source: pointBuy
+                amount: -2
+            }
+        ]
+        temporary.modifiers: [
+            Bonus {
+                name: size.name
+                amount: sizeBonus(size.value)
+            }
+        ]
+    }
+
+    Skills {
+
+    }
+
+    AbilityScore {
+        id: dexterity
+        name: "Dexterity"
+
+        permanent.modifiers: [
+            Bonus {
+                source: pointBuy
+                amount: 6
+            },
+            Bonus {
+                source: race
+                amount: 4
+            }
+        ]
+        temporary.modifiers: [
+            Bonus {
+                name: size.name
+                amount: -sizeBonus(size.value)
+            }
+        ]
+    }
+
+    AbilityScore {
+        id: constitution
+        name: "Constitution"
+
+        permanent.modifiers: [
+            Bonus {
+                source: pointBuy
+                amount: 4
+            }
+        ]
+    }
+
+    AbilityScore {
+        id: intelligence
+        name: "Intelligence"
+
+        permanent.modifiers: [
+            Bonus {
+                source: pointBuy
+                amount: 4
+            },
+            Bonus {
+                source: race
+                amount: 2
+            }
+        ]
+    }
+
+    AbilityScore {
+        id: wisdom
+        name: "Wisdom"
+
+        permanent.modifiers: [
+            Bonus {
+                source: pointBuy
+                amount: 2
+            }
+        ]
+    }
+
+    AbilityScore {
+        id: charisma
+        name: "Charisma"
+
+    }
+
+
+
+
+    Item {
+        id: hp
+        property string prefix: "attr://combat/hitPoints"
+
+        Attribute {
+            id: maxHp
+            name: "Maximum Hit Points"
+            uri: hp.prefix + "/maximum"
+
+            modifiers: [
+                Bonus {
+                    name: constitution.temporary.modifier.name
+                    amount: constitution.temporary.modifier.value*level.value
+                },
+                Bonus {
+                    name: "Level 1 Health Roll (Max)"
+                    amount: 8
+                },
+                Bonus {
+                    name: "Favoured Class"
+                    amount: level.value
+                },
+                Bonus {
```

Comment:
You can probably use the shared Toughness feat. 
Look at the changes I made to fernie/Character.qml in #18 to see how to use it.